python/python/glide/async_commands/server_modules/ft.py

```diff
@@ -78,6 +81,42 @@ async def dropindex(client: TGlideClient, indexName: TEncodable) -> TOK:
     return cast(TOK, await client.custom_command(args))
 
 
+async def search(
+    client: TGlideClient,
+    indexName: TEncodable,
+    query: TEncodable,
+    options: Optional[FtSeachOptions],
+) -> List[Union[int, Mapping[TEncodable, Mapping[TEncodable, TEncodable]]]]:
```

Comment:
Can we use tuple? That will simplify usage and testing.

Reply:
I would prefer List because it is an array in the documentation. Also, List is consistent across languages.
Also, the complexity in testing is inside Mapping and not at the list level.